benchmarks/quant-hela-method-comparison/scripts/04_compute_metrics.py

```
@@ -0,0 +1,581 @@
+    Returns correlation statistics and raw values for plotting.
+    """
+    tmt_df = load_quantification_result("PXD007683-TMT", method)
+    lfq_df = load_quantification_result("PXD007683-LFQ", method)
+
+    if tmt_df is None or lfq_df is None:
+        return None
+
+    intensity_col = INTENSITY_COLUMNS.get(method)
+
+    # Find intensity columns
+    for df_name, df in [("TMT", tmt_df), ("LFQ", lfq_df)]:
```

Ruff (0.14.14)

[warning] Loop control variable `df_name` not used within loop body

Rename unused `df_name` to `_df_name`

(B007)